lib/sqlalchemy/orm/exc.py

```diff
@@ -196,9 +200,7 @@ def _default_unmapped(cls):
 
     try:
         mappers = base.manager_of_class(cls).mappers
-    except NO_STATE:
-        mappers = {}
-    except TypeError:
+    except _multi_exc(TypeError, *NO_STATE):
```

Comment:
In python 3 we could just do `except (TypeError, *NO_STATE):` but it does not work on python 2.

I would suggest `except (TypeError,) + NO_STATE:` instead of adding a function

Reply:
I figured the function would allow the code to read as much as the Python 3 equivalent (modulo the literal function name there), but if dropping it better fits the practice within the project, I'm happy to follow that.

Reply:
I did not think of that.  My suggestion what it was with the + it's immediately clear what it does :)